### PR TITLE
dashboard part 2: Show all the nodes' status on the dashboard 

### PR DESCRIPTION
**Pull request by ping-ke**
issue: https://github.com/ethstorage/es-node/issues/253
Added grafana dashboard to overview the network states

### How it works
Add --state.upload.url http://65.109.63.154:8080 to run.sh, and the node will upload the state to metrics service every minute. And the whole network states will show in grafana [ethstorage-network](https://grafana.ethstorage.io/d/a043a1fd-0e8f-42cc-ab9f-5e92a3f2bb35/ethstorage-network?orgId=2&refresh=5m).

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -102,6 +102,7 @@ es_node_start=" --network devnet \
   --l1.beacon-based-time 1706684472 \
   --l1.beacon-based-slot 4245906 \
   --download.thread 32 \
+  --state.upload.url http://65.109.63.154:8080 \
```
Comment (qzhodl) [inline run.sh:105]: Do we want to bind a subdomain to the server to make changing the IP more convenient

**Comment by qzhodl** [comment]
I would suggest splitting this PR into several smaller PRs for better code review.

File: run.sh (hunk 1 above)
Comment (qzhodl) [reply run.sh:126]: http://metrics.ethstorage.io has been mapped to 65.109.63.154

File: ethstorage/metrics/network_metrics.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	return &NetworkMetrics{
+
+		PeersTotal: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Subsystem: SubsystemName,
+			Name:      "peers",
+			Help:      "The number of peers existed in the last 10 minutes",
+		}),
+
+		MinersOfShards: factory.NewGaugeVec(prometheus.GaugeOpts{
```
Comment (qzhodl) [inline ethstorage/metrics/network_metrics.go:55]: Where does this data show in the dashboard?

**Review by syntrust — APPROVED** [review]

**Review by qzhodl — APPROVED** [review]

**Pull request merged**